lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -365,6 +393,23 @@ def va_profile_opt_in_out_lambda_handler(  # noqa: C901
                     }
                 )
 
+        va_profile_id = bio['vaProfileId']
+
+        # Send Comp and Pen Opt-In confirmation if anticipated status code still 200
+        # And if Opt-In confirmation (bio['allowed'] == True)
+        if post_response['statusCode'] == 200 and bio['allowed']:
+            response = send_comp_and_pen_opt_in_confirmation(va_profile_id)
+
+            # Save notification_id from POST sms response if method returned
+            # a value AND the response status code is 201.
+            if response is not None and response.status == 201:
+                response_data = response.read().decode()
+                response_json = json.loads(response_data)
+                notification_id = response_json['id']
+                save_notification_id_to_cache(va_profile_id, notification_id, bio['sourceDate'])
+            else:
+                logger.critical('Could not send Comp and Pen opt-in confirmation to VAProfileId: %s', va_profile_id)
```

Comment:
don't we want to log what the POST v2/notifications/sms post response is here if we get something other than a 201 from our API?

Reply:
Updated.